conftest.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import os
+from selenium import webdriver
+
+try:
+    from utils.variable import global_web_driver as driver
+except ImportError:
+    from .utils.variable import global_web_driver as driver
+
+
+@pytest.fixture(scope="function", autouse=True)
+def my_fixture(request):
```

Comment:
In real projects it will be good to use more detailed fixture name, e.g. web_driver(). But for test project my_fixture() is ok)